packages/data-point/lib/entity-types/factory.test.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-env jest */
+
+const { createEntityFactory } = require('./factory')
+
+describe('creating a model with createEntityFactory', () => {
+  const factory = createEntityFactory('model')
+  const spec = { value: input => input }
+  const instance = factory('nuts', spec)
```

Comment:
we might want a test that actually creates/resolves an entity from this "instance"